frontend/api/pairing/index.ts

```diff
@@ -0,0 +1,12 @@
+import { PairingRequest } from "@/api/pairing/types";
+
+export function getPairingServiceUri(pairingRequest: PairingRequest) {
+  return `ws://localhost:4000/pairing?user=${pairingRequest.userId}`;
+}
+
+// TODO: remove harcoded values
+const PAIRING_SERVICE = "ws://localhost:4000/pairing";
```

Comment:
Remove the hardcoded values, add the port and url to the .env instead. Include information on the port number to use somewhere so we can use it

Reply:
Other comments:
1. set up docker compose file in the pairing services
2. add config of the port to use for pairing in the docker compose file, probably taking the values from environment variables.
3. add simple explanation of pairing-front and pairing-back to README to understand their function.
4. Add instructions in the pairing services on how to run them (like port details and all that) Right now its quite unclear how they are connected to, and how they are supposed to be run